### ISSUE TITLE
Unable to run rspec tests due to foreign key constraint

### ISSUE THREAD

**Issue by jeremygoh** [MarkUsProject/Markus#2250]
Foreign key constraints means we are unable to truncate the tables in pg as part of our rspec configuration.

```
/home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/activerecord-4.2.3/lib/active_record/connection_adapters/postgresql/database_statements.rb:155:in `async_exec': PG::FeatureNotSupported: ERROR:  cannot truncate a table referenced in a foreign key constraint (ActiveRecord::StatementInvalid)
DETAIL:  Table "groupings" references "groups".
HINT:  Truncate table "groupings" at the same time, or use TRUNCATE ... CASCADE.
: TRUNCATE groups
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/activerecord-4.2.3/lib/active_record/connection_adapters/postgresql/database_statements.rb:155:in `block in execute'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/activerecord-4.2.3/lib/active_record/connection_adapters/abstract_adapter.rb:473:in `block in log'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/activesupport-4.2.3/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/activerecord-4.2.3/lib/active_record/connection_adapters/abstract_adapter.rb:467:in `log'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/activerecord-4.2.3/lib/active_record/connection_adapters/postgresql/database_statements.rb:154:in `execute'
	from /home/vagrant/Markus/spec/spec_helper.rb:61:in `block (3 levels) in <top (required)>'
	from /home/vagrant/Markus/spec/spec_helper.rb:61:in `each'
	from /home/vagrant/Markus/spec/spec_helper.rb:61:in `block (2 levels) in <top (required)>'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/lib/rspec/core/example.rb:378:in `instance_exec'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/lib/rspec/core/example.rb:378:in `instance_exec'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/lib/rspec/core/hooks.rb:357:in `run'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/lib/rspec/core/configuration.rb:1660:in `block in run_hooks_with'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/lib/rspec/core/configuration.rb:1660:in `each'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/lib/rspec/core/configuration.rb:1660:in `run_hooks_with'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/lib/rspec/core/configuration.rb:1626:in `with_suite_hooks'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/lib/rspec/core/runner.rb:114:in `block in run_specs'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/lib/rspec/core/reporter.rb:77:in `report'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/lib/rspec/core/runner.rb:113:in `run_specs'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/lib/rspec/core/runner.rb:89:in `run'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/lib/rspec/core/runner.rb:73:in `run'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/lib/rspec/core/runner.rb:41:in `invoke'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/gems/rspec-core-3.3.2/exe/rspec:4:in `<top (required)>'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/bin/rspec:23:in `load'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/bin/rspec:23:in `<main>'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/bin/ruby_executable_hooks:15:in `eval'
	from /home/vagrant/.rvm/gems/ruby-2.1.3@markus/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by jeremygoh** [comment]
I'll submit a fix shortly

**Comment by david-yz-liu** [comment]
Closed in #2251.
